src/envoy/mixer/send_attribute_filter.cc

```diff
@@ -0,0 +1,116 @@
+/* Copyright 2016 Google Inc. All Rights Reserved.
```

Comment:
Please change copy right to 2017 Istio Authors. That's what we are doing in manager.